flake.nix

```diff
@@ -78,20 +83,31 @@
 
               services.postgres = {
                 enable = true;
+                package = pkgs.postgresql_15;
                 initialScript = ''
                   ${builtins.readFile ./dev/postgres/00-setup.sql}
 
                   CREATE USER postgres SUPERUSER PASSWORD 'postgres';
                 '';
                 listen_addresses = "127.0.0.1";
                 port = 6432;
+                settings = {
+                  max_prepared_transactions = 262143;
```

Comment:
It is maximal possible value, in tests I have set it to `2000` as it doesn't matter really as long as it is greater than `0` I think.